src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField-Attached.Edit.cshtml

```diff
@@ -0,0 +1,63 @@
+@model OrchardCore.Media.ViewModels.EditMediaFieldViewModel
+@using OrchardCore.Media.Settings;
+@using OrchardCore.ContentManagement.Metadata.Models
+
+@{
+    var settings = Model.PartFieldDefinition.Settings.ToObject<MediaFieldSettings>();
+    var mediaFieldId = Html.IdFor(m => m);
+}
+
+<partial name="Shared/MediaFieldEditResources.cshtml" />
+
+<script at="Foot">
+
+    initializeAttachedMediaField(
+        document.getElementById('@mediaFieldId'),
+        '@mediaFieldId-field-file-upload',
+        '@Url.Action("Upload", "Admin", new { area = "OrchardCore.Media" }) ',
```

Comment:
Don't inject `@` inside javascript. Create hidden divs that have `data` attributes with the value, and load it from javascript.